platform/security/core/security-core-impl/pom.xml

```diff
@@ -149,6 +217,14 @@
                 <configuration>
                     <instructions>
                         <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
+                        <Embed-Dependency>
```

Comment:
You should definitely not do both. I would lean towards removing the private package statements as those are a little more dangerous than embedding jars. It looks like we are embedding guava and including the com.google.* packages. Same is true for the ddf.security.common stuff. I'm also surprised that you need to embed the validation-api? Usually you don't embed apis because the point of them is that their interfaces are available throughout the osgi runtime. The correct approach would probably be to include the validation-api bundle (if you look at it, it is already a bundle because that is how it is meant to be used) in the feature that includes security-core. A good rule of thumb is that we don't embed bundles, we embed jars.